docs/v2.5/1_getting-started.md

```diff
@@ -25,12 +25,22 @@ This page will walk you through the process of setting up a project ID and using
 
 ### Install via Open UPM
 
-1. Open up your project in editor
-2. Follow the OpenUPM guide [here](https://openupm.com/docs/getting-started.html) to install openupm-cli
-3. Navigate to your Unity Project folder and run the commands ```openupm add io.chainsafe.web3-unity``` & ```openupm add io.chainsafe.web3-unity.web3auth```
-4. Import Samples from Package Manager web3.unity Package Page
+1. Navigate to Window/PackageManager on the top bar in unity.
+2. Add a new scoped registry or edit the existing OpenUPM entry if it's already there.
```

Comment:
They should add the new scoped registry, if they edit the existing one, it will not recognize the package that you've replaced (I think).

Reply:
the guide says editing is fine, im assuming it refreshes the asset db after